<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerWindowQWidget</class>
 <widget class="QWidget" name="TimerWindowQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tasker - Session</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QWidget" name="TaskContainerQWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="taskQLabel">
        <property name="text">
         <string>Task:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="taskQComboBox">
        <item>
         <property name="text">
          <string>Writing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Compose Music</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="startTimerQPushButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="backQPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Perferences</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="taskListenersQGroupBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Listeners</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QComboBox" name="audioQComboBox">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>121</width>
           <height>25</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Built-In Mic</string>
          </property>
         </item>
        </widget>
        <widget class="QCheckBox" name="audioQCheckBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>50</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Mic</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="keyboardQCheckBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>83</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Keyboard</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="productionTimeAlertQGroupBox">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>100</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Others</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Notify me when done</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="productionTimerQWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>160</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="productionTimeDescriptionsQLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Hours</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="productionTimeHoursQSpinBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Hours</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Minutes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="productionTimeQLabel">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>15</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Goal</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="productionTimeMinutesQSpinBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Minutes</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="prefix">
         <string>:</string>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>true</bool>
        </property>
        <property name="prefix">
         <string>:</string>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
